client/src/js/services/DocumentService.js

```diff
@@ -0,0 +1,40 @@
+angular.module('bhima.services')
+.service('DocumentService', DocumentService);
+
+DocumentService.$inject = [ '$http', 'util' ];
+
+function DocumentService($http, util) {
```

Comment:
What do you think of exposing this service under the `PatientService`?  Something like:
```js
angular.module('bhima.services')
.service('PatientService', PatientService);

PatientService.$inject = [ /* injects ... */, 'DocumentService'];

function PatientService(/* imports .... */, Documents) {
  var service = this;

  // .. service definition...

  service.Documents = Documents;

   // ... more code ...
}
```
That way, a controller would only import `PatientService` and have access to `DocumentService` by default?  What do you think?